packages/desktop-client/src/components/transactions/SelectedTransactions.jsx

```diff
@@ -135,6 +135,8 @@ export function SelectedTransactionsButton({
           case 'link-schedule':
             pushModal('schedule-link', {
               transactionIds: [...selectedItems],
+              getTransaction : getTransaction,
+              pushModal: pushModal,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I'm surprised linter didn't pick this up
```suggestion
              getTransaction,
              pushModal,
```

Reply:
Adjusted as per suggestions, 
Is object notation not Key Value, 
I am not normally a JS/TS developer so to me this looks weird.